NeoMathEngine/src/MathEngineDnnDropout.cpp

```diff
@@ -34,7 +34,21 @@ static inline int getMaskSize( float rate, bool isSpatial, bool isBatchwise, con
 	return batchWidth * objectSize;
 }
 
-CMathEngineDropoutDesc::CMathEngineDropoutDesc( IMathEngine& mathEngine, float rate, bool isSpatial, bool isBatchwise,
+static inline int getCacheSize(float rate, bool isSpatial, bool isBatchwise, const CBlobDesc& input)
+{
+	const int cacheSize = 64;
+	const int maskSize = getMaskSize(rate, isSpatial, isBatchwise, input);
+	if(isSpatial) {
+		const int maskAlign = 4;
```

Comment:
Move `static constexpr int maskAlign = 4;` into the class CSeedDropoutDesc, to use it here